stable_baselines/trpo_mpi/utils.py

```diff
@@ -111,10 +106,11 @@ def traj_segment_generator(policy, env, horizon, reward_giver=None, gail=False):
         cur_ep_ret += rew
         cur_ep_true_ret += true_rew
         current_it_len += 1
+        current_ep_len += 1
         if done:
             ep_rets.append(cur_ep_ret)
             ep_true_rets.append(cur_ep_true_ret)
-            ep_lens.append(current_it_len)
+            ep_lens.append(current_ep_len)
```

Comment:
`current_ep_len` should be resetted here, no?

Reply:
This is true! Sorry about that, I'll add that and changes to the changelog.